docs/Getting Started.md

```diff
@@ -0,0 +1,43 @@
+# Clone Serai
+
+```
+git clone https://github.com/serai-dex/serai.git
+```
+
+# Build and Run Serai
+
+- Ubuntu 18.04 native or WSL2 w/ Docker (Other distributions with docker may also work)

```

Comment:
ubuntu 18.04?? a bit of an older distro lol

Reply:
Calling me old?? Swapped to LTS.